pkg/app/app.go

```diff
@@ -221,6 +222,30 @@ func Convert(opt kobject.ConvertOptions) {
 
 	komposeObject.Namespace = opt.Namespace
 
+	// convert env_file from absolute to relative path
+	for _, service := range komposeObject.ServiceConfigs {
+		if len(service.EnvFile) <= 0 {
+			continue
+		}
+		for i, envFile := range service.EnvFile {
+			if !filepath.IsAbs(envFile) {
+				continue
+			}
+
+			workDirAbs, err := filepath.Abs(filepath.Dir(opt.InputFiles[0]))
+			if err != nil {
+				log.Fatalf(err.Error())
+			}
+
+			relPath, err := filepath.Rel(workDirAbs, envFile)
+			if err != nil {
+				log.Fatalf(err.Error())
+			}
+
+			service.EnvFile[i] = strings.Replace(relPath, "\\", "/", -1)
```

Comment:
We should be using https://pkg.go.dev/path/filepath#FromSlash not replacing `\\`.

Using fromSlash returns OS-specific path names.

Reply:
It doesn't solve the issue...
I need unix path and it still returns unix path

example:
```golang
package main

import (
	"fmt"
	"path/filepath"
)

func main() {
	p := filepath.FromSlash("C:\\test\\main.go")
	fmt.Println(p)
}
```
output:
```text
C:\test\main.go
```